lib/plugins/extension/helper/list.php

```diff
@@ -294,29 +311,41 @@ public function makeLegend(helper_plugin_extension_extension $extension, $showin
      */
     public function makeLinkbar(helper_plugin_extension_extension $extension)
     {
-        $return  = '<div class="linkbar">';
-        $return .= $this->makeHomepageLink($extension);
-        if ($extension->getBugtrackerURL()) {
-            $return .= ' <a href="'.hsc($extension->getBugtrackerURL()).
-                '" title="'.hsc($extension->getBugtrackerURL()).'" class ="bugs">'.
-                $this->getLang('bugs_features').'</a> ';
+        global $conf;
+        $html  = '<div class="linkbar">';
+        $html .= $this->makeHomepageLink($extension);
+
+        $bugtrackerURL = $extension->getBugtrackerURL();
+        if ($bugtrackerURL) {
+            $param = array(
+                'href'   => $bugtrackerURL,
+                'title'  => $bugtrackerURL,
+                'class'  => 'bugs',
+                'target' => $conf['target']['extern'],
```

Comment:
I noticed that bugtracker url won't check if the link is on www.dokuwiki.org. I searched through the plugin repo, and the only plugin that has a www.dokuwiki.org bugtracker is [plugin:workflow](https://www.dokuwiki.org/plugin:workflow) - should be fine though.

Reply:
@phy25 Thanks for the comment. The bugtracker url will be checked, see f11bfd3